knowledge-base/changing-edit-mode-behavior-maui-raddatagrid.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Changing Edit Mode Behavior in RadDataGrid for MAUI
+description: Learn how to modify the edit mode behavior in RadDataGrid for MAUI to enter edit mode with a single click instead of double-clicking.
```

Comment:
```suggestion
description: Learn how to modify the edit mode behavior in RadDataGrid for MAUI to enter edit mode with a single click instead of a double-click.
```